File: mcconaha-steve/century-from-year.js
/*Introduction
The first century spans from the year 1 up to and including the year 100, the second century 
from the year 101 up to and including the year 200, etc.

Task
Given a year, return the century it is in.*/

function century(year) {
    // FUNCTION
    // IF year is less than 101
    if (year < 101) {
    // THEN century === 1
      return 1;
    } else {
      return (Math.ceil(year / 100));
    // ELSE century === year / 100 rounded up to the nearest integer
      console.log(Math.ceil(year / 100));
    }
  };
   // ENDFUNCTION

File: mekonnen-naod/century-from-year.py
# use conditional: if year < 1000 return year // 100 + 1 
# if year >= 1000 and year % 100 == 0, return year/ 100
# else return year // 100 + 1

def century(year):
    if year < 1000:
        return year // 100 + 1
    elif year >= 1000 and year % 100 == 0:
        return year / 100
    else:
        return year // 100 + 1
